src/custom/state/user/reducer.ts

```diff
@@ -102,6 +102,9 @@ function _initialStatePerChain(chainId: number) {
   )
 }
 
+// TODO: fix SDK enum build
+const ALL_SUPPORTED_CHAIN_IDS = [SupportedChainId.MAINNET, SupportedChainId.GNOSIS_CHAIN, SupportedChainId.GOERLI]
+
```

Comment:
What's wrong here?

Reply:
Actually nothing, removed TODO, thank you!